guides/doc-Administering_Red_Hat_Satellite/topics/uploading-extra-scap-content.adoc

```diff
@@ -1,15 +1,30 @@
-[id='uploading-extra-scap-content_{context}']
+[id="Uploading_Extra_SCAP_Content_{context}"]
```

Comment:
I personally prefer `this-id-convention` but I am good. We will probably never be able to fix everything.

Reply:
Well, don't underestimate my love for consistency 😂  Only 118 instances to go.

Reply:
I wrote this BEFORE I saw part1 and part2 anchors PRs :-)